### DISCUSSION TITLE
Curious Question: Where did the name mozc originate from?

### DISCUSSION THREAD

**Discussion by My1** [google/mozc#943] [General]
to be frank until recently I always associated mozc with mozilla, and only learned recently that it is actually based on Google's JP IME, although a lot of things clicked for me as it felt really natural to use for me (after getting it to work in the first place lol), after having used it on windows since the ages of XP (back then for a lack of a JP IME on non-JP windows to my knowledge back then) all the way through windows 8 (as I did not really like MS's IME) and apparently now in Linux too lol.

The Linux Version is even better than Windows as you can select your base keyboard layout of choice (in my case german-nodeadkeys) which is really awesome with special characters and stuff.

although to be frank as an avid follower of mozc-devices I seriously should have noticed the connection earlier (thanks for making me laugh year after year).

so I am kinda curious, where does the name come from or what does it mean?

**Comment by ammgws** [comment]
This page has an answer: https://developers-jp.googleblog.com/2012/04/android-google-1410193.html

**Reply by tats-u** [reply]
It doesn't mention why they chose the names of those see vegetables.
